Users/Install.rst

```diff
@@ -159,7 +159,14 @@ can run
 
 ::
 
-    (venv)$ pip3 install coala-bears --pre
+    (venv)$ git clone https://github.com/coala/coala
+    (venv)$ cd coala/coala
+    (venv)$ pip3 install .
+    (venv)$ cd -
+    (venv)$ git clone https://github.com/coala/coala-bears
+    (venv)$ cd coala/coala-bears
+    (venv)$ pip3 install .
```

Comment:
Ok, I'm not comfortable with this command block. Since this is intended for non-devs who may want to know what's going on, you should explain each step. 
Also, I'm not sure this is the best way to document this. Please open an issue so we can discuss this.